wiki/Tournaments/o!mLN/4/en.md

```diff
@@ -0,0 +1,387 @@
+| Sheeter | ::{ flag=US }:: [EpsilonMaiagare](https://osu.ppy.sh/users/3855052), ::{ flag=CN }:: [Yuiesta](https://osu.ppy.sh/users/13953619) |
+| GFX Designer | ::{ flag=SE }:: [Melancholicc](https://osu.ppy.sh/users/18406374), ::{ flag=IT }::  [MightyMigz](https://osu.ppy.sh/users/8863387), ::{ flag=ID }:: [nayuu](https://osu.ppy.sh/users/12561379), ::{ flag=PH }:: [Normiplier](https://osu.ppy.sh/users/10069850), ::{ flag=PH }:: [OsuMe65](https://osu.ppy.sh/users/852867), ::{ flag=SG }:: [Polytetral](https://osu.ppy.sh/users/8612061), ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024), ::{ flag=PH }:: [Swarmsii](https://osu.ppy.sh/users/11716111), ::{ flag=PH }:: [yukina meng](https://osu.ppy.sh/users/9919550), ::{ flag=MY }:: [Z419](https://osu.ppy.sh/users/9912966) |
+| Developer | ::{ flag=KR }:: [Nopekjk](https://osu.ppy.sh/users/4585186), ::{ flag=KR }:: [yhsphd](https://osu.ppy.sh/users/6665667) |
+| Composer | ::{ flag=TH }:: [HowToPlayLN](https://osu.ppy.sh/users/10879600) |
+
+## Links
+
+- [Registrations news post](https://osu.ppy.sh/home/news/2025-07-21-osu-mania-ln-tournament-4-registrations-now-open)
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/2107840?n=1)
+- [Challonge Bracket](https://challonge.com/5whqliol)
+- [o!mLN Discord Sever](https://discord.gg/SPQtt3s)
+- [Livestream](https://www.twitch.tv/osumanialive)
+- **[Information spreadsheet](https://docs.google.com/spreadsheets/d/1Wl3Q41AI0e36sINVWh6qJHtKcCG_fdju2_0CdBm0DGs/edit?usp=sharing)**
+- **[Statistics sheet](https://drive.google.com/drive/folders/1sBl4ej38-oYWs6Wc7al-KMskdfkmZilu)**
+
+## Ordeal in Illusion
+
+![Ordeal in Illusion banner](img/ordeal-in-Illusion.png "The Ordeal in Illusion banner in forum post")
+
+**Ordeal In Illusion** is a leaderboard event that runs tangent to the tournament's schedule, available for everyone to participate!
+
+After mappool showcase every week starting the Round of 64, **original songs released in the week's mappool** will receive a leaderboard on the Statistics sheet, allowing both players and spectators to compete for the best score on our originals.
+
+Before the start of the next round's mappool showcase, leaderboards for the week's originals will be frozen. Tickets are then awarded to every participant who submitted scores to the leaderboard, with the number of tickets awarded increasing with their ranking on the leaderboard:
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 5 Tickets |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 Tickets |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 3 Tickets |
+| *4-7th place* | 2 Tickets |
+| *Other participates* | 1 Ticket |
+
+At the end of the tournament, Tickets earned by all participants are tabulated and 3 Tickets will be drawn during a livestream. The owners of these Tickets will win a part of the community-donated OII prize pool!
```

Comment:
Don't understand what "tabulated" is making reference to. As in counting the tickets from each player? If so, the suggestion below should work better.
```suggestion
At the end of the tournament, tickets earned by all participants are tallied and 3 tickets will be drawn during a livestream. The owners of these tickets will win a part of the community-donated OII prize pool!
```